sep490-frontend/src/app/modules/shared/pipes/translate-params.pipe.ts

```
@@ -0,0 +1,32 @@
+    return this.translate.instant(
+      value,
+      typeof params === 'object' ? this.translateParams(params) : undefined
+    );
+  }
+
+  private translateParams(
+    params: Record<string, string | number | boolean>
+  ): Record<string, string | number | boolean> {
+    return Object.keys(params).reduce(
+      (parameters, key) => ({
+        ...parameters,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)